requirements/base.txt

```diff
@@ -2,4 +2,18 @@ Django>=4.1,<4.2
 django-dotenv==1.4.1
 wagtail>=4.1,<5
 wagtailfontawesome>=1.1.3,<1.2
+dj-database-url==1.0.0
 django-debug-toolbar>=3.2,<4
+elasticsearch==5.5.3
+# Additional dependencies for Heroku, AWS, and Google Cloud deployment
```

Comment:
Why have these been moved from production.txt? We don't really want users to have to install these packages if they're just setting up a local build, especially if they have native code dependencies (as psycopg2 does).

Reply:
I think I was a little trigger-happy when moving these dependencies. I agree a few of them aren't necessary during local development (`psycopg2` being a good example of one which can be a pain on windows).

Reply:
With that said, whilst vagrant didn't install the production dependencies, the docker install did, as did the virtualenv install path, so even before they weren't very distinct.